index.js

```diff
@@ -104,12 +110,25 @@ function parseMembers(options) {
 
       if (customBaseIri && split[1].startsWith(customBaseIri.prefix + ':')) {
         latestCustomVocabElementId = split[1];
-        customVocab.push({
-          '@id': latestCustomVocabElementId,
-          '@type': 'Class',
-          'subClassOf': {'@id': 'schema:Thing'}
-        });
+          customVocab.push({
+            '@id': latestCustomVocabElementId,
+            '@type': 'Class',
+            'subClassOf': {'@id': 'owl:Thing'}
+          });
+      }
+    } else if (split[0] === '@superTypes') { // always assumes @type has already been defined
+      // able to define extra classes beyond 'type' to create class hierarchy
+      let subClasses = [];
+      for (let i = 1, size = split.length; i < size ; i++) {
+        subClasses.push(split[i]);
       }
+        
+      customVocab.pop() // remove previous default subClass definition
```

Comment:
Why?

Reply:
Because the @type attribute already adds a default superclass in case the shape in question pertains to the custom base iri. This superclass info has to be replaced with the new info defined through @superTypes. Possibly there is a more elegant way of doing this.

Reply:
Ok, makes sense. The `pop` will not work if `@type` is defined after `@superTypes`. We can't assume any order here.

Reply:
Indeed, the order is important. I might be reading it wrong, but doesn't @extraTypes currently also presuppose a strict ordering? The line "types.push(idToType[id]);" seems to presuppose the earlier line "idToType[id] = split[1];" which only happens when @type has already been encountered.

Reply:
Yeah, you're right. In that case you can assume the same. Also add it to the README then so users are aware.

Reply:
Will do!